riscv_config/constants.py

```diff
@@ -17,8 +30,10 @@
         "Zpn", "Zpsf",
         "Zmmul",
         "Svnapot"
-]
+] + Zvl_extensions + Zve_extensions
+
 Zb_extensions = ["Zba", "Zbb", "Zbc", "Zbe", "Zbf", "Zbm", "Zbp", "Zbr", "Zbs", "Zbt"]
 Zp_extensions = ["Zbpbo", "Zpn", "Zpsf"]
```

Comment:
While you're at it, could you also do the same for these two (`Zp_extensions` and `Zb_extensions`) as you did for the ones you added?
I.e. remove their entries from `Z_extensions` and instead append the lists like you did at line +33 🙂

Reply:
Done.